lib/aurora_bootstrapper/table.rb

```diff
@@ -1,11 +1,16 @@
+require 'aws-sdk-s3'
+
 module AuroraBootstrapper
   class Table
-    def initialize( database_name:, table_name:, client:, blacklisted_fields: [] )
+    def initialize( database_name:, table_name:, client:, blacklisted_fields: [], s3_client: nil )
       @blacklisted_fields = blacklisted_fields
       @database_name      = database_name
       @table_name         = table_name
       @client             = client
-      @export_date        = ENV.fetch( 'EXPORT_DATE', nil )
+      d                   = DateTime.now
+      d_str               = d.strftime("%Y-%m-%d")
+      @export_date        = ENV.fetch( 'EXPORT_DATE', d_str )
```

Comment:
are these variables used anywhere?

Reply:
At line 12.

Reply:
I will move this part of code to exporter.rb and pass @export_date around to different other files to make sure we put dump files under the same datetime folder